ui/src/app/components/apps-panel/expanded-app.spec.tsx

```diff
@@ -42,6 +42,7 @@ import { ALL_GKE_APP_STATUSES, minus } from 'testing/utils';
 
 import { ExpandedApp } from './expanded-app';
 import { UIAppType } from './utils';
+import {PROXY_PATH_SUFFIX} from "../../utils/constants";
```

Comment:
the linter's not going to happy about this: run `eslint` in IntelliJ or `yarn lint --fix` at the CLI